invenio_communities/members/records/api.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Northwestern University.
+#
+# Invenio-Communities is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from invenio_db import db
+from invenio_records.systemfields import ConstantField, ModelField
+from invenio_records_resources.records.api import Record
+from invenio_records_resources.records.systemfields import IndexField
+
+from .models import MemberModel
+
+
+class Member(Record):
```

Comment:
Same comment here as for invitations in that it should not be a record but simply a sqlalchemy model (similar to record/community tables. I

Reply:
After implementing permissions (https://github.com/inveniosoftware/invenio-communities/pull/394), having Member be a Record actually seems like a good thing. Let's discuss this again perhaps?